app/controllers/api/v1/materials_controller.rb

```diff
@@ -50,7 +38,21 @@ def material_json_params
     params.permit(material_json_schema)
   end
 
-  def includes
+  def included_relations_to_render
     [:material_type, :metadata]
   end
+
+  def filter(params)
+    resources = Material
```

Comment:
I think, this could go to a Mixin (maybe a concern) and the resource would be a parameter.